sllurp/llrp.py

```diff
@@ -631,7 +639,7 @@ def set_reader_config_cb(state, is_success, *args):
             if self.disconnecting:
                 return
             self.send_ENABLE_EVENTS_AND_REPORTS()
-            self.send_SET_READER_CONFIG(onCompletion=set_reader_config_cb)
+            self.send_SET_READER_CONFIG(self.llrp_keepalive_interval_ms, onCompletion=set_reader_config_cb)
```

Comment:
As you have directly access to the interval_ms value from config, there is no need to change to change the function prototype here and to have to pass it through.